release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -71,7 +71,7 @@ See xref:../installing/installing_azure_stack_hub/installing-azure-stack-hub-use
 [id="ocp-4-9-upgrade-pause-mhc"]
 ==== Pausing machine health checks before updating the cluster
 
-During the upgrade process, nodes in the cluster might become temporarily unavailable. In the case of worker nodes, the machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, {product-title} {product-version} introduces `cluster.x-k8s.io/paused=""` annotation to let you pause the `MachineHealthCheck` resources before updating the cluster. 
+During the upgrade process, nodes in the cluster might become temporarily unavailable. In the case of worker nodes, the machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, {product-title} {product-version} introduces `cluster.x-k8s.io/paused=""` annotation to let you pause the `MachineHealthCheck` resources before updating the cluster.
```

Comment:
Should this be 

'. . . introduces the `cluster.x. . . ' ? 

With an article? Other instances in the docs have an article. Just curious.

Reply:
Cc: @abhatt-rh